packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -77,48 +76,56 @@ import {
 import Seo from '@salesforce/retail-react-app/app/components/seo'
 import {Helmet} from 'react-helmet'
 
-const onClient = typeof window !== 'undefined'
-
-/*
-The categories tree can be really large! For performance reasons,
-we only load the level 0 categories on server side, and load the rest
-on client side to reduce SSR page size.
-*/
-const useLazyLoadCategories = () => {
-    const itemsKey = 'categories'
-
-    const levelZeroCategoriesQuery = useCategory({
-        parameters: {id: CAT_MENU_DEFAULT_ROOT_CATEGORY, levels: CAT_MENU_DEFAULT_NAV_SSR_DEPTH}
-    })
-
-    const ids = levelZeroCategoriesQuery.data?.[itemsKey]?.map((category) => category.id)
-    const queries = useCategoryBulk(ids, {
-        enabled: onClient && ids?.length > 0
-    })
-    const dataArray = queries.map((query) => query.data).filter(Boolean)
-    const isLoading = queries.some((query) => query.isLoading)
-    const isError = queries.some((query) => query.isError)
-    return {
-        isLoading,
-        isError,
-        data: {
-            ...levelZeroCategoriesQuery.data,
-            [itemsKey]: mergeMatchedItems(
-                levelZeroCategoriesQuery.data?.categories || [],
-                dataArray
-            )
-        }
+const PlaceholderComponent = () => (
+    <Center p="2">
+        <Spinner size="lg" />
+    </Center>
+)
+
+const DrawerMenuItem = withCommerceSdkReact(
```

Comment:
Mimicking the variable name `ListMenuContentWithData`:
```suggestion
const DrawerMenuItemWithData = withCommerceSdkReact(
```